app/src/main/java/com/example/foody/recipe_search/presentation_mvvm/ui/SearchScreen.kt

```diff
@@ -153,11 +193,9 @@ private fun RecipeItem(item: RecipeInfo, goToDetailsScreen: (String) -> Unit) {
         Column(
             modifier = Modifier
                 .padding(8.dp)
-                .clickable(
-                    enabled = true,
-                    // klikom na recept idemo na detalje
-                    onClick = { goToDetailsScreen(item.id) }
-                )
+                .clickable(enabled = true,
```

Comment:
It was more properly formatted before.

Reply:
I'm not sure what you mean. I'll take a look...

Reply:
I'm referring to the formatting: `enabled = true` on new line, `)` after `}` at end also on new line.